.changeset/rare-parents-press.md

```diff
@@ -0,0 +1,5 @@
+---
+"@sap/guided-answers-extension-webapp": patch
```

Comment:
Hi @Klaus-Keller, shouldn't changeset include patch update of all the modules?

Reply:
@hitesh-parmar, this should take care of it:
https://github.com/SAP/guided-answers-extension/blob/84f6c1da168eac06dc0ada5d9f8c4fc651f7dc6f/.changeset/config.json#L5

Reply:
Thanks Klaus

Reply:
Hi @Klaus-Keller , Looks like in this case changelog entry is not created https://github.com/SAP/guided-answers-extension/blob/main/packages/ide-extension/CHANGELOG.md